packages/@aws-cdk/aws-codepipeline/lib/action.ts

```diff
@@ -51,6 +51,8 @@ export interface ActionBind {
    * The IAM Role to add the necessary permissions to.
    */
   readonly role: iam.IRole;
+
+  readonly actionRole?: iam.IRole;
```

Comment:
Docs

Reply:
I actually made this module-private, as it should have been from the beginning, so no need for docs anymore.